package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.9.2",
+  "version": "0.9.2-d1",
```

Comment:
versions shouldn't be bumped in PRs.
```suggestion
  "version": "0.9.2",
```

Reply:
Sorry for tampering the branch. Clean now